### PR TITLE
Enrich markdown sample with more features showcase.

### PR DESCRIPTION
**Pull request by halilozercan**

### CODE REVIEW

File: docs/richtext-commonmark.md [hunk 1]
```diff
@@ -16,31 +16,40 @@ dependencies {
...
+      * Unordered sub-list.
+  3. And another item.
+      You can have properly indented paragraphs within list items. Notice the blank line above, and the leading spaces (at least one, but we'll use three here to also align the raw Markdown).
+
+  * Unordered list can use asterisks
+  - Or minuses
+  + Or pluses
+  ---
+
+  ```javascript
```
Comment (LouisCAD) [inline docs/richtext-commonmark.md:37]: Why not a Kotlin example? 😋

File: docs/richtext-commonmark.md [hunk 2]
```diff
@@ -16,31 +16,40 @@ dependencies {
 The simplest way to render markdown is just pass a string to the [`Markdown`](/api/com.zachklipp.richtext.markdown/-markdown/)
 composable:
 
-```kotlin
+~~~kotlin
```
Comment (zach-klippenstein) [inline docs/richtext-commonmark.md:19]: Are these tildes to make nested fenced blocks work?

**Review by zach-klippenstein — CHANGES_REQUESTED** [review]
Please make the image the same width as the other screenshots, 384px. Also it would be great if you could add the status bar and navigation bar just to frame the screen a little bit, like the other screenshots. If not that’s fine too though.

File: docs/richtext-commonmark.md (hunk 1 above)
Comment (zach-klippenstein) [reply docs/richtext-commonmark.md:37]: We don’t currently support syntax highlighting, so it doesn’t make too much of a difference. But I’d like to make that possible. I think this is something that should probably be solved at the markdown level, eg maybe an additional parameter on the ‘Markdown’ composable that is a composable function that takes the some kind of code block ast abstraction and can render it however it wants.

File: docs/richtext-commonmark.md (hunk 2 above)
Comment (halilozercan) [reply docs/richtext-commonmark.md:19]: exactly

**Comment by halilozercan** [comment]
I'm taking the screenshots on my phone because can't run emulator on my AMD pc right now :/ Unfortunately, rolling screenshots don't include navigation bar but this example should do the job.

**Comment by zach-klippenstein** [comment]
Ah, didn't realize it was rolling. I typically use gifs for that but I don't feel strongly about the details as long as it's about the same width as the others.

**Review by zach-klippenstein — APPROVED** [review]

**Pull request merged**